definition-map.js

```diff
@@ -193,6 +193,73 @@ const MUTATIONS = {
             completedAssessmentAttemptId: "The ID of the last completed assessment attempt.",
         }
     },
+    CreateThread: {
+        metadescription: "Creates a thread of comments under a commentable type.",
+        args: {
+            forumId: "The ID of the forum.",
```

Comment:
I wouldn't think you'd have to manually add these as they should be covered by this block, right?

https://github.com/thoughtindustries/helium-graphql/blob/57b346dae70b4244efb6611efc8eabe724ecedc2/augment-schema.js#L83-L85